zkg.meta

```diff
@@ -3,6 +3,7 @@ script_dir = analyzer
 plugin_dir = build/spicy-modules
 
 build_command = mkdir -p build && cd build && SPICYZ=%(package_base)s/spicy-plugin/build/plugin/bin/spicyz cmake .. && make -j 2
+#build_command = mkdir -p build && cd build && cmake .. && make -j
```

Comment:
Looks like left-over?

Reply:
This won't compile without that line for me, so I just left it in there as convenience.  I can remove.